projects/demo/src/modules/components/data-list/examples/2/index.html

```diff
@@ -69,7 +69,7 @@
         </tui-data-list>
     </ng-template>
     <ng-template #drinksTmp>
-        <tui-data-list>
+        <tui-data-list [tuiActiveZoneParent]="activeZone">
```

Comment:
Is it really necessary? I think it is only needed if this data list has submenus.

Reply:
fixed